README.md

```diff
@@ -1,140 +1,75 @@
 
-You can pass mutually exclusive gesture sets in a constructor of `AndroidGestureManager` or with `AndroidGestureManager#setMutuallyExclusiveGestures()`.
+## Documentation
 
-Example:
-```
-    Set<Integer> mutuallyExclusive1 = new HashSet<>();
-    mutuallyExclusive1.add(AndroidGesturesManager.GESTURE_TYPE_SHOVE);
-    mutuallyExclusive1.add(AndroidGesturesManager.GESTURE_TYPE_SCROLL);
-
-    Set<Integer> mutuallyExclusive2 = new HashSet<>();
-    mutuallyExclusive2.add(AndroidGesturesManager.GESTURE_TYPE_SHOVE);
-    mutuallyExclusive2.add(AndroidGesturesManager.GESTURE_TYPE_SCALE);
-
-    AndroidGesturesManager androidGesturesManager = new AndroidGesturesManager(
-      context,
-      mutuallyExclusive1,
-      mutuallyExclusive2
-    );
-```
+You'll find all of this library's documentation on [our Mapbox Gestures page](https://www.mapbox.com/android-docs/gestures/overview). This includes information on installation, using the API, and links to the API reference.
 
-The first set makes it certain that when we detect shove, we will no longer be notified about scroll (shove will be able to execute because scroll is not a `ProgressiveGesture`).
-The second, that when we detect either shove or scale we won't be notified about the other one until the first gesture finishes.
 
-#### Thresholds
-You can set thresholds for supported gestures, which means that gesture detector won't fire until the threshold (like minimum rotation angle) is met. This allows you to personalize gestures experience however you like.
+## Getting Started
 
-We encourage to set thresholds using `dimen` values, rather than raw pixels, to accommodate for various screen sizes and pixel densities across Android devices. For example:
+If you are looking to include Mapbox Gestures for Android inside of your project, please take a look at [the detailed instructions](https://www.mapbox.com/android-docs/map-sdk/overview/gestures/) found in our docs. If you are interested in building from source, read the contributing guide inside of this project.
 
-```
-androidGesturesManager.getStandardScaleGestureDetector()
-.setSpanSinceStartThresholdResource(R.dimen.scaleSpanSinceStartThreshold);
-```
-and for thresholds that are not expressed in pixels:
-```
-androidGesturesManager.getRotateGestureDetector().setAngleThreshold(ROTATE_ANGLE_THRESHOLD);
+To use the Gestures library, include it in your app-level `build.gradle` file.
+
+```java
+// In the root build.gradle file
+repositories {
+    mavenCentral()
+}
 ```
 
-#### Velocity
-Each progressive gesture with its respective `#onEnd()` callback will provide `X velocity` and `Y velocity` of the gesture at the moment of pointers leaving the screen.
+```java
+// In the app build.gradle file
+dependencies {
+    implementation 'com.mapbox.mapboxsdk:mapbox-android-plugin-offline:0.1.0'
```

Comment:
This should rather be `'com.mapbox.mapboxsdk:mapbox-gestures-android:0.2.0'`.

Reply:
Yikes. Fixed